src/base/orgs/View.svelte

```diff
@@ -141,33 +120,32 @@
 {:then org}
   {#if org}
     <main>
+      {#await Profile.get(address, config)}
+        <Loading fadeIn />
+      {:then profile}
       <header>
         <div class="avatar">
-          {#if registration && registration.avatar}
-            <img src={registration.avatar} alt="avatar" />
-          {:else}
-            <Blockies address={org.address} />
-          {/if}
+          <Avatar source={profile.avatar() ?? address} />
         </div>
         <div class="info">
           <span class="title bold">
-            {registration ? label : ethers.utils.getAddress(address)}
+            <Address resolve noAvatar {address} {config} />
```

Comment:
Are we resolving twice here? Since we already have the profile but the Address is having to do the lookup again? I think this is a case where we should use a conditional, or pass an optional `label` to Address. Not sure which is a better idea..

Reply:
I think to avoid introduce noise into the `<Address>` component we should check i.e.: `profile.name() ?? address` and remove the `<Address>` component here.